.github/workflows/build.yml

```diff
@@ -4,7 +4,7 @@ on: [ push, pull_request ]
 
 jobs:
   linux:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
so, `ubuntu-latest` should be `ubuntu 20.04` soon anyway [if it isn't already];
but it is usually the LTS that most CIs use;